webpack.config.js

```diff
@@ -323,6 +323,7 @@ let appConfig = {
     ...(!IS_CI
       ? [
           new ForkTsCheckerWebpackPlugin({
+            eslint: true,
```

Comment:
Can we make this based on an env var so that it can be disabled?

Reply:
@priscilawebdev Ah, I was thinking there should also be an env var to specifically disable `eslint` e.g. I want to fork the ts checker, but I don't want ts checker to run eslint.